lib/reply.js

```diff
@@ -539,19 +539,30 @@ function handleError (reply, error, cb) {
     return
   }
 
-  const serializerFn = getSchemaSerializer(reply.context, statusCode)
-  const payload = (serializerFn === false)
-    ? serializeError({
-        error: statusCodes[statusCode + ''],
-        code: error.code,
-        message: error.message || '',
-        statusCode: statusCode
-      })
-    : serializerFn(Object.create(error, {
-      error: { value: statusCodes[statusCode + ''] },
-      message: { value: error.message || '' },
-      statusCode: { value: statusCode }
-    }))
+  let payload
+  try {
+    const serializerFn = getSchemaSerializer(reply.context, statusCode)
+    payload = (serializerFn === false)
+      ? serializeError({
+          error: statusCodes[statusCode + ''],
```

Comment:
Just a question, why not template literals here? Something related to performance or just standard style?

Reply:
Habit I guess. I've never checked the performance of them for converting a number to string.

Reply:
I personnally did some experiments on fast-json-stringify and those are pretty equivalent